frontend/packages/dev-console/integration-tests/features/user-preferences/user-preferences-dev-perspective.feature

```diff
@@ -0,0 +1,92 @@
+@odc-5227
+Feature: Manage user preferences
+              As a user, I need the ability to set/edit my preferences for the OCP Console.
+
+
+        Background:
+            Given user is at developer perspective
+
+
+        @regression @to-do
+        Scenario: Setting Developer preference for perspective: UP-01-TC01
+            Given user in at admin perspective
+             When user clicks on user dropdown on masthead
+              And user clicks User Preference option
```

Comment:
I would recommend to have one action here so that you can select the item within this dropdown. I believe this makes it easier to implement the action later, and imho it describes more what the user is doing:

```suggestion
             When user clicks on user dropdown and selects "User Preference"
```

Reply:
Updated